### PR TITLE
Remove units and people edits layers

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/19398

## Testing

**URL to test:** 
Local

**Steps to test:**
1. Check out this branch, get a fresh replica of prod data. Apply hasura migrations and metadata.
2. Lets test a user edit to the units table:
```sql
select vin from units where id = 1;
select vin from units_cris where id = 1;
update units set vin = 'VZ EDIT' where id = 1;
select vin from units where id = 1;
```
3. Lets test a cris update to that same record, the unified record should show the VZ edit:
```sql
update units_cris set vin = 'CRIS UPDATE' where id = 1;
select vin from units where id = 1;
select vin from units_cris where id = 1;
```
4. Lets test a cris update to a non edited value, the unified record should show the new cris value:
```sql
select vin from units where id = 2;
select vin from units_cris where id = 2;
update units_cris set vin = 'CRIS UPDATE' where id = 2;
select vin from units where id = 2;
select vin from  units_cris where id = 2;
```
5. Now lets test a user null override. The unified record should show null instead of the cris value
```sql
select vin from units where id = 10;
select vin from units_cris where id = 10;
update units set vin = null where id = 10;
select vin from units where id = 10;
select vin from units_cris where id = 10;

-- extra test - reset the unified value to match the _cris schema
update units set vin = '1FMCU22XXVUC99371' where id = 10;
-- edit the cris value
update units_cris set vin = 'CRIS EDIT' where id = 10;
-- observe that the unified value is now back in sync with the cris value
select vin from units where id = 10;
```
6. Lets test a user edit to the people table:
```sql
select prsn_first_name from people where id = 1;
select prsn_first_name from people_cris where id = 1;
update people set prsn_first_name = 'VZ EDIT' where id = 1;
select prsn_first_name from people where id = 1;
```
7. Lets test a cris update to that same record, the unified record should show the VZ edit:
```sql
update people_cris set prsn_first_name = 'CRIS UPDATE' where id = 1;
select prsn_first_name from people where id = 1;
select prsn_first_name from people_cris where id = 1;
```
8. Lets test a cris update to a non edited value, the unified record should show the new cris value:
```sql
select prsn_first_name from people where id = 2;
select prsn_first_name from people_cris where id = 2;
update people_cris set prsn_first_name = 'CRIS UPDATE' where id = 2;
select prsn_first_name from people where id = 2;
select prsn_first_name from  people_cris where id = 2;
```
9. Now lets test a user null override. The unified record should show null instead of the cris value
```sql
select prsn_first_name from people where id = 198611;
select prsn_first_name from people_cris where id = 198611;
update people set prsn_first_name = null where id = 198611;
select prsn_first_name from people where id = 198611;
select prsn_first_name from people_cris where id = 198611;

-- extra test - reset the unified value to match the _cris schema
update people set prsn_first_name = 'ADAM' where id = 198611;
-- edit the cris value
update people_cris set prsn_first_name = 'CRIS EDIT' where id = 198611;
-- observe that the unified value is now back in sync with the cris value
select prsn_first_name from people where id = 198611;
```
10. Next we can test the cris import script. You can download some extract zips from the AWS S3 bucket and put them in `/vision-zero/etl/cris_import/extracts` then follow the test steps in  `cris_import/README.md` to run the cris import docker image and run the following command to process those extract zips:
```shell
./cris_import.py --csv
```
11. You can also inspect the units and people change logs and make sure you are seeing your changes there as expected
12. Test the down migrations

---

#### Ship list

- [ ] Check migrations for any conflicts with latest migrations in `main` branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: database/migrations/default/1728066705364_drop_crashes_edits_update_crashes_cris_functions/down.sql [hunk 1]
```diff
@@ -1,7 +1,3 @@
---
--- Name: change_log_crashes_edits; Type: TABLE; Schema: public; Owner: visionzero
---
-
```
Comment (roseeichelmann) [inline database/migrations/default/1728066705364_drop_crashes_edits_update_crashes_cris_functions/down.sql:4]: just getting rid of the annoying pg dump comments in this migration

File: database/migrations/default/1728873263563_drop_units_edits_layer/up.sql [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    SELECT to_jsonb(units) INTO unified_record_jb from public.units where public.units.id = new.id;
+
+    -- for every key in the cris json object
+    for column_name in select jsonb_object_keys(new_cris_jb) loop
+        -- ignore audit fields
+        continue when column_name in ('created_at', 'updated_at', 'created_by', 'updated_by');
+        -- if the new value doesn't match the old
+        if(new_cris_jb -> column_name <> old_cris_jb -> column_name) then
+            -- see if the unified record has the same value as the old cris value
+            if (unified_record_jb -> column_name = old_cris_jb -> column_name) then
```
Comment (roseeichelmann) [inline database/migrations/default/1728873263563_drop_units_edits_layer/up.sql:127]: this is the line that is changed ^

File: database/migrations/default/1728873263563_drop_units_edits_layer/down.sql [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+        new.veh_damage_severity2_id,
+        new.veh_hnr_fl,
+        new.veh_make_id,
+        new.veh_mod_id,
+        new.veh_mod_year,
+        new.veh_trvl_dir_id,
+        new.vin
+    );
+    -- insert new (editable) vz record (only record ID)
+    INSERT INTO public.units_edits (id) values (new.id);
```
Comment (roseeichelmann) [inline database/migrations/default/1728873263563_drop_units_edits_layer/down.sql:285]: this is the line thats removed

**Review by johnclary — APPROVED** [review]
i tested this a lot and, yep, it's working perfectly! 

FYI that I opened [an issue](https://github.com/cityofaustin/atd-data-tech/issues/19534) to update the OCR narrative ETL because it references the `_edits` layer.

🚢

**Review by mddilley — APPROVED** [review]
All the test steps showed the expected results when updating CRIS values, updating edits, and nulling over CRIS values too. I read through the code, and the changes to the people and unit triggers makes sense to me. The import and down migrations work as well. Looks great! 😎

**Pull request merged**